docs/raising-events.md

```diff
@@ -34,20 +34,33 @@ robot.FellInLove += Raise.WithEmpty(); // sender will be robot, args will be Eve
 robot.FellInLove += Raise.With(sender: robot, e: someEventArgs);
 ```
 
-Events of type `EventHandler<TEventArgs>` can be raised in exactly the same way. 
+Events of type `EventHandler<TEventArgs>` can be raised in exactly the same way.
 
-Note that as of version 3.1.0, for the .NET Standard version of the FakeItEasy library, `TEventArgs` need not extend `EventArgs` to support the above syntax. The .NET 4.0 library still requires `TEventArgs` to extend `EventArgs`.
+Note that as of version 3.1.0, for the .NET Standard and .NET 4.5 versions of the FakeItEasy library, `TEventArgs` need not extend `EventArgs` to support the above syntax. The .NET 4.0 library still requires `TEventArgs` to extend `EventArgs`.
 
-If an event is defined using a **custom delegate**, then `Raise` needs
-a typeparam to help it out:
+It is also possible to raise events defined using a **custom delegate** (a.k.a
+"free-form delegate"), like these:
 
 ```csharp
 public delegate void CustomEventHandler(object sender, CustomEventArgs e);
 public delegate void FreeformEventHandler(int count);
 …
 event CustomEventHandler CustomEvent;
 event FreeformEventHandler FreeformEvent;
-…
+```
+
+To do that, you can either:
+
+* use `Raise.FreeForm.With`, which automatically infers the correct delegate type:
+
+```csharp
+fake.CustomEvent += Raise.FreeForm.With(fake, sampleCustomEventArgs);
+fake.FreeformEvent += Raise.FreeForm.With(7);
+```
+
+* specify the delegate type explicitly as a type parameter to `Raise.With`:

```

Comment:
I think this would read better with an initial "or": 

* or specify the delegate type…